devices/debian.py

```diff
@@ -18,7 +18,7 @@ class DebianBox(base.BaseDevice):
     A linux machine running an ssh server.
     '''
 
-    prompt = ['root\\@.*:.*#', '/ # ', ]
+    prompt = ['root\\@.*:.*#', '/ # ', ".*:~ #" ]
```

Comment:
This change should probably be in a separate commit/pull request but I don't think it should be an issue this time.